src/value-converters/abstract-locale-value-converter.ts

```diff
@@ -21,11 +29,21 @@ export abstract class AbstractLocaleValueConverter {
         }
     }
 
+    protected setLocale(locale:string) {
+        AbstractLocaleValueConverter._locale = locale;
+    }
+
+    protected getLocale() {
+        return AbstractLocaleValueConverter._locale;
+    }
+
     constructor(
         private readonly _eventAggregator:EventAggregator,
     ) {
+        this.setLocale(AbstractLocaleValueConverter.getDefaultLocale());
```

Comment:
So, we set the locale in instance one and then create instance two... boom!
... Or, looking a few lines down, we just set the global locale after creating a specific instance for a different locale - boom!
... Or, we set the i18n locale and then create instance n+1... boom!

This is so _unnecessarily_ vicious!